src/main/java/frc/robot/subsystems/swervev2/SwervePosEstimator.java

```diff
@@ -39,6 +50,9 @@ public SwervePosEstimator(GenericEncodedSwerve frontLeftMotor, GenericEncodedSwe
                         backRightMotor.getPosition(),
                 },
                 new Pose2d());
+        NetworkTableInstance inst = NetworkTableInstance.getDefault();
```

Comment:
where is all the code to add vision to poseEstimator? Is this in a different branch?